microsoft-support.md

```diff
@@ -0,0 +1,43 @@
+# Microsoft support for grpc-dotnet
+
+> gRPC is a CNCF project and receives support from the gRPC team and the grpc-dotnet maintainers and contributors. All gRPC languages officially supported by CNCF have this level of support.
```

Comment:
Is the `>` markdown format ("blockquote") a good way to render a note like this?

Reply:
What would you prefer? The docs.microsoft.com system allows for special formatting of notes, but basic markdown is more limited.

Reply:
Would italics or bold+italics look better?

Reply:
Would italics or bold+italics look better?

Reply:
Done